e2e/test_operator_ha_upgrades/operator_ha_upgrade_test.go

```diff
@@ -381,4 +381,38 @@ var _ = Describe("Operator HA Upgrades", Label("e2e"), func() {
 		EntryDescription("Upgrade from %[1]s to %[2]s with network link that drops some packets"),
 		fixtures.GenerateUpgradeTableEntries(testOptions),
 	)
+
+	DescribeTable(
+		"Test ha cluster generation number during upgrade",
```

Comment:
We will merge the changes/checks from here into `upgrading a cluster with operator pod chaos and without foundationdb pod chaos`

Reply:
Done.